sdks/java/io/elasticsearch/OWNERS

```diff
@@ -4,3 +4,4 @@ reviewers:
   - echauchot
   - jbonofre
   - timrobertson100
+  - egalpin
```

Comment:
you should put your name first IMHO otherwise very little chance to be pinged as reviewer